client/me/purchases/paths.js

```diff
@@ -1,53 +1,35 @@
 /** @format */
-function purchasesRoot() {
-	return '/me/purchases';
-}
 
-function addCreditCard() {
-	return purchasesRoot() + '/add-credit-card';
-}
+export const purchasesRoot = '/me/purchases';
```

Comment:
It seems there is an occurrence of `purchasesRoot` that still calls it as a function in https://github.com/Automattic/wp-calypso/blob/master/client/me/purchases/index.js#L119 that needs to be updated too.

Currently, it'll cause an error (to reproduce, you can visit the legacy `/purchases` route: http://calypso.localhost:3000/purchases)

Reply:
Thanks! That was sloppy of me - updated in 6de976f800